### DISCUSSION TITLE
Works on my machine, how can I thank you?

### DISCUSSION THREAD

**Discussion by zwindler** [jahnf/Projecteur#194] [Ideas]
I work for a tech company and I waited a long time to decide whether or not to buy a Logictech Spotlight because it didn't support Linux.

With default deb package and `--disable-uinput`, it works out of the box on Ubuntu 22.04.1 + wayland.

Conferences and presentations are a part of my job and this tool will help me. I'd like to thank you and the team working on it. How can I do that?

**Comment by jahnf** [comment]
Hello, thank you for the feedback!

I  don't have time too much for the project currently, since priorities are on mostly on different projects and my day job at the moment. 
That said: Feedback for every working and non-working configuration is helpful - same goes for workarounds and information that help other uses. First so it is documented here for others to find and when I come back to the project I can integrate information to the docs and work much better on open issues for the 1.0 of the project which I still plan to release at some point :)

No special thanks needed, feedback on working configurations, bugs and also pull-requests are enough - If you have a blog or other social media accounts a post on how you use your pointing device with `Projecteur` on Linux is also welcomed very much.

**Comment by zwindler** [comment]
That I can do.

I'll write a little blog post on my personal tech blog and share the repository on socials.

Thanks again :)
